pywbem/cim_operations.py

```diff
@@ -861,6 +890,100 @@ def __init__(self, url, creds=None, default_namespace=DEFAULT_NAMESPACE,
         self._last_server_response_time = None
 
     @property
+    def operation_recorder(self):
+        """
+        This is a writeable property; setting this property will cause the
+        defined recorder to be activated.
+
+        Effective pywbem 0.10.0 operation_recorder became a property rather
+        than an attribute. The getter operation returns TODO
+        The setting operation executes the add_operation_recorder with the
+        value provided. Since there can be multiple simultaneous recorders,
+        setting operation_recorder to None is no longer valid.
+
+
+        **Experimental:** This property is experimental for this release.
+
+        **Deprecated:** This property has been marked deprecated.  The method
+        add_operation_recorder should be used to activate an operation
+        recorder.
+        """
+        warnings.warn(
+            "Directly setting operation recorder has been deprecated",
+            DeprecationWarning)
+        return self._operation_recorder
+
+    @operation_recorder.setter
+    def operation_recorder(self, value):
+        """
+        Set a the value representing the operation recorder instance
+        into the list of active operation recorders.
+
+        Parameters:
+           value ()
+              Instance of a subclass of the Recorder class
+
+        Exceptions:
+            ValueError if value is None.
+
+        **Experimental:** This property is experimental for this release.
+
+
+        """
+        self._operation_recorder = value
+        if value is None:
+            raise ValueError('WBEMConnection operation recorder set to None'
+                             ' not allowed.')
+        self.add_operation_recorder(value)
+        warnings.warn(
+            "Directly setting operation recorder has been deprecated",
+            DeprecationWarning)
+
+    @property
+    def operation_recorder_enabled(self):
+        """
+        :class:`py:bool`: operation_recorder enablement status for connection.
+
+        This is a writeable property; setting this property will change the
+        operation recorders enablement status accordingly for all active
+        operation recorders.
+
+        This property returns None if no operation recorder has been added.
+
+        If any recorder in the recorders list has been enabled, it returns
+        true. Otherwise it returns false
+
+        **Experimental:** This property is experimental for this release.
+        """  # noqa: E501
+
+        for recorder in self._operation_recorders:
+            if recorder.enabled():
+                return True
+
+        return False
+
+    @operation_recorder_enabled.setter
+    def operation_recorder_enabled(self, value):
+        """
```

Comment:
Again, this docstring is not being generated into the documentation.

Reply:
Moved the parameter to the property and put note in doc for setter to see property